subsys/nrf_rpc/nrf_rpc_uart.c

```diff
@@ -22,6 +22,17 @@ enum {
 	HDLC_CHAR_DELIMITER = 0x7e,
 };
 
+enum flip_state {
+	FLIP_ZERO,
+	FLIP_ONE,
+	FLIP_ANY
+};
+
+struct trx_flips {
+	uint8_t tx_flip : 2;
+	uint8_t rx_flip : 2;
```

Comment:
For TX, `FLIP_ANY` has no interpretation so it might be more self-explanatory to use sth like:
```suggestion
	uint8_t tx_flip : 1;
	uint8_t rx_flip : 1;
	bool rx_flip_valid : 1;
```

Reply:
The main idea of `FLIP_ANY` is to not lose the first received frame since receiver doesn't have background data. Validness bit doesn't solve the problem. Despite `FLIP_ANY` doesn't have sense on tx side, I'd suggest keeping it `as is` for equality. Do not see big difference here between 1 or 2 bits since both are packed well in the shortest possible type.

Reply:
1 bit is not a big difference, of course :). I just was confused just looking at the type definition why we need `FLIP_ANY` for TX and thought this could be made clearer. But it's fine, feel free to disregard this comment.